src/main/java/com/example/springbootapp/controller/EmployeeController.java

```diff
@@ -20,27 +25,32 @@ public EmployeeController(EmployeeService employeeService) {
     }
 
     @GetMapping
-    public List<EmployeeDTO> getAllEmployees(){
+    public List<EmployeeDTO> getAllEmployees() {
+        LOGGER.info(employeeService.getAllEmployees().toString());
         return employeeService.getAllEmployees();
     }
 
     @GetMapping("/{employeeId}")
-    public EmployeeDTO getById(@PathVariable("employeeId") long employeeId){
+    public EmployeeDTO getById(@PathVariable("employeeId") long employeeId) {
+        LOGGER.info(employeeService.getEmployeeById(employeeId).toString());
```

Comment:
i added logback file with configuration which represents information about the type of request and input params. But there is a problem which I don't know how to solve. Logs string ends like that:"ge (truncated)..." because it too long, and I all of my tries to solve it failed:(

Reply:
It's enough for now. Let's discuss it after